src/main.rs

```diff
@@ -1,3 +1,6 @@
+mod ast;
```

Comment:
I guess we can re-organise the sources a little later once we finish up all the stuff with parsing.

Reply:
Yeah we probably wanna create separate crates for some stuff.